content/get_start/UseCase_Workflows/pollination.md

```diff
@@ -0,0 +1,75 @@
+
+   - Apart from Extruded Floors option for geometry calculation, Detailed option for the window calculation method was selected. 
+   - The model was visualized to make sure there are no issues. To avoid having minor issues after the export such as gaps between floors and adjacency issues that can eventually cause the EnergyPlus simulation to fail, the model was previewed in plane and wireframe view as well and following steps were carried out. 
+      - The height was adjusted to close any gaps formed in the model due to the default heights. 
+      - The Align option was used to specify Reference Planes and Model Lines in Revit to pull the room/space boundaries to the nearest plane/line within a specified Align Distance defined in Pollination. This can help clean and fix the export without actually modifying the Revit model. 
+      - Solve Adjacency was set to Full. 
+
+![userdata](/BEM-for-PRM/get_start/UseCase_Workflows/images/DD_Extr_Step6_After_Fix.png?width=700px&align=right&classes=border,alignCenter)
+
+##### **2. Define the Construction Set, Program type, HVAC**
+Details related to "Construction Set", "Program Type" and "HVAC" were specified within Pollination.
+    
+  - [Construction Sets](https://docs.pollination.cloud/user-manual/rhino-plugin/energy-modeling/assign-construction-sets): Several options for construction templates with different vintages (based on ASHRAE 90.1 Baseline constructions in OpenStudio Standards) are available, and can also be edited. For this workflow, 2019 Vintage, Climate Zone 5 and Steel Framed type was selected. 
+
+  - [Program Types](https://docs.pollination.cloud/user-manual/rhino-plugin/energy-modeling/assign-program-types): Several options for load definitions and schedules for people, lighting, equipment, infiltration, ventilation, setpoint, service hot water corresponding to a list of space types and different vintages, are available and can be edited. Here, 2019 Vintage, with Medium Office as the Building Type and three program types - Conference, OpenOffice and Closed Office were selected accordingly for different spaces. 
+
+  - [HVAC](https://docs.pollination.cloud/user-manual/rhino-plugin/energy-modeling/assign-hvac-systems): Only "Ideal air loads" option is available by default and hence was selected (which was modeled as zone equipment). However, an HVAC system is a key element to run the PRM measure, thus, HVAC was revisited after exporting the model to OpenStudio. 
+    There is also an option in Pollination to import HVAC systems from OpenStudio that could be considered to assign HVAC systems. 
+
+  - Service Hot Water (SHW): There is an option for SHW but no system to be selected.
+
+  - Space Names: To avoid having random handlers assigned to the space names and thermal zones after the export, "Reset Identifier" option was enabled in Pollination's final step before export. This option typically enables Pollination to parse the correct space names into the OpenStudio model. 
```

Comment:
Is this OpenStudio object handlers? We may want to make a clarification.
`having random OpenStudio object handlers`

Reply:
@AmmarDeh Can you please confirm the above before I add it?

Reply:
Done! 
@weilixu Based on Ammar's comment, if you want to suggest anything else, I'll update it again.

Reply:
Please follow Ammar's suggestion.